.github/workflows/buildapp.yml

```diff
@@ -0,0 +1,189 @@
+# Original idea by @ISnackable. Thanks to him for handling the hardest parts!
```

Comment:
Maybe credit the uYouEnhanced script up here in a comment as well, since they made this base script haha.